### PR TITLE
Feature/models for map sync locks

### PR DESCRIPTION
**Pull request by deevezina**

### CODE REVIEW

File: src/Components/MapSyncRequestModel.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="MapSyncRequestModel.cs" company="Talegen">
```
Comment (zeroadam) [inline src/Components/MapSyncRequestModel.cs:2]: GlobalLink Vasont

File: src/Components/MapSyncRequestModel.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+//     Copyright (c) 2017 Talegen. All rights reserved.
```
Comment (zeroadam) [inline src/Components/MapSyncRequestModel.cs:3]: 2020 GlobalLink Vasont

File: src/Components/MapSyncResponseModel.cs [hunk 3]
```diff
@@ -0,0 +1,24 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="MapSyncResponseModel.cs" company="Talegen">
```
Comment (zeroadam) [inline src/Components/MapSyncResponseModel.cs:2]: GlobalLink Vasont

File: src/Components/MapSyncResponseModel.cs [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+//     Copyright (c) 2017 Talegen. All rights reserved.
```
Comment (zeroadam) [inline src/Components/MapSyncResponseModel.cs:3]: 2020 GlobalLink Vasont

File: src/Components/MapSyncRequestModel.cs (hunk 1 above)
Comment (deevezina) [reply src/Components/MapSyncRequestModel.cs:2]: what the heck? didn't notice that. It's how they are in Framework

**Pull request merged**